Add heatit brand (#43391)
home-assistant/home-assistant.github.io: source/_integrations/heatit.markdown

---
title: Heatit

> description: Connect and control your Heatit Zwave devices using the Zwave integration

Reviewer: ```suggestion
description: Connect and control your Heatit Z-Wave devices using the Z-Wave integration
```

ha_release: '2026.2'
ha_iot_class: Local Push
ha_domain: heatit
ha_integration_type: brand
ha_iot_standard: zwave
ha_brand: true
---

> [Heatit](https://www.heimantech.com/) is committed to making sure their products are up-to-date and ready to use in Home Assistant.

Reviewer 2: This url is not correct.

> [Heatit](https://www.heimantech.com/) is committed to making sure their products are up-to-date and ready to use in Home Assistant.

Reviewer: ```suggestion
[Heatit](https://heatit.com/) is committed to making sure their products are up-to-date and ready to use in Home Assistant.
```
Is this the product website?

> Heatit Zwave devices work locally and integrate seamlessly with the Zwave.js integration in Home Assistant. As all connectivity is happening locally, status updates and controlling your devices happen instantly in Home Assistant.

Reviewer: ```suggestion
Heatit Z-Wave devices work locally and integrate seamlessly with the Z-Wave.js integration in Home Assistant. Because all connectivity occurs locally, status updates and device control happen instantly in Home Assistant.
```

> {% my add_matter_device badge domain=page.ha_domain %}

Reviewer 2: This is not a matter integration.

> {% my add_matter_device badge domain=page.ha_domain %}

Reviewer: ```suggestion
{% my add_zwave_device badge domain=page.ha_domain %}
```

> [Learn more about Zwave in Home Assistant.](/integrations/zwave_js/)

Reviewer: ```suggestion
[Learn more about Z-Wave in Home Assistant.](/integrations/zwave_js/)
```

---
title: Heatit
description: Connect and control your Heatit Z-Wave devices using the Z-Wave integration

> ha_release: '2026.2'

Reviewer: ```suggestion
ha_release: 2026.2
```

ha_iot_class: Local Push
ha_category:
  - Climate
  - Sensor
  - Binary sensor

>   - Sensor

Reviewer 2: ```suggestion
```

  - Switch
ha_domain: heatit
ha_integration_type: brand
works_with:
  - zwave
ha_platforms:
  - climate
  - sensor
  - binary_sensor

>   - sensor

Reviewer 2: ```suggestion
```
